testify/test_runner.py

```diff
@@ -31,6 +30,10 @@
 __author__ = "Oliver Nicholas <bigo@yelp.com>"
 __testify = 1
 
+EX_OK = 0  # successful termination
+EX_DATAERR = 65  # EX_DATAERR -- The input data was incorrect in some way.
+EX_SOFTWARE = 70  # EX_SOFTWARE -- An internal software error has been detected.
```

Comment:
also no reason to repeat the constant name in the comment about the constant name, I think it's fairly obvious what you are referring to :)